rqt_py_common/src/rqt_py_common/rosaction.py

```diff
@@ -363,34 +374,39 @@ def rosactionmain(mode=MODE_ACTION):
     except KeyboardInterrupt:
         pass
 
+
 """
 "#NOT_TESTED_FROM_HERE"----------------------------------------
 From here are what are copied from __init__.py that I don't know yet
 if they are necessary/useful.
 """
- ## copied from the web, recipe for ordered yaml output ######
+## copied from the web, recipe for ordered yaml output ######
+
+
 def construct_ordered_mapping(self, node, deep=False):
     if not isinstance(node, yaml.MappingNode):
         raise yaml.constructor.ConstructorError(None, None,
-                "expected a mapping node, but found %s" % node.id,
-                node.start_mark)
+                                                "expected a mapping node, but found %s" % node.id,
+                                                node.start_mark)
     mapping = collections.OrderedDict()
     for key_node, value_node in node.value:
         key = self.construct_object(key_node, deep=deep)
         if not isinstance(key, collections.Hashable):
             raise yaml.constructor.ConstructorError("while constructing a mapping",
                                                     node.start_mark,
-                    "found unhashable key", key_node.start_mark)
+                                                    "found unhashable key", key_node.start_mark)
```

Comment:
break for fourth argument

Reply:
Not actually required for flake8 or pep8